compute_tools/src/spec.rs

```diff
@@ -758,6 +758,14 @@ BEGIN
     END LOOP;
 END $$;
 "#,
+        r#"
+DO $$
```

Comment:
should you also add the migration for already created `neon_superuser`s?

Reply:
I THINK they should inherit it, since we turn inherit on for them in earlier migrations. I will test in staging though

Reply:
maybe you’re right, but let’s check

Reply:
Ok I tested locally. First is before granting neon_superuser the role and second one is after (censored password/IP address)
```
sasha=> CREATE SUBSCRIPTION sub CONNECTION 'postgresql://sasha:xxx@yyy/neondb' PUBLICATION pub;
ERROR:  permission denied to create subscription
DETAIL:  Only roles with privileges of the "pg_create_subscription" role may create subscriptions.
sasha=> CREATE SUBSCRIPTION sub CONNECTION 'postgresql://sasha:xxx@yyy/neondb' PUBLICATION pub;
ERROR:  could not connect to the publisher: connection to server at "yyy" (ipipip), port 5432 failed: ERROR:  connection is insecure (try using `sslmode=require`)
```